fastestimator/network.py

```diff
@@ -13,11 +13,12 @@
 # limitations under the License.
 # ==============================================================================
 from collections import ChainMap
-from typing import Any, Callable, Dict, Iterable, List, MutableMapping, Set, Union
+from typing import Any, Callable, Dict, Iterable, List, MutableMapping, Optional, Set, Union
```

Comment:
you added the Optional import, but I don't see it being used in any of the relevant places (like List[Optional[str]] for weight path)

Reply:
done